README.rst

```diff
@@ -2,8 +2,12 @@
 zc.intid - Reduced-conflict integer id utility
 ==============================================
 
-.. image:: https://travis-ci.org/zopefoundation/zc.intid.svg?branch=master
-    :target: https://travis-ci.org/zopefoundation/zc.intid
+.. image:: https://github.com/zopefoundation/zc.intid/actions/workflows/tests.yml/badge.svg
+    :target: https://github.com/zopefoundation/zc.intid/actions/workflows/tests.yml
+
+.. image:: https://readthedocs.org/projects/zcintid/badge/?version=latest
+        :target: http://zcintid.readthedocs.io/en/latest/?badge=latest
```

Comment:
shouldn't this be a http**s** url ? (also I'm not sure the `?badge=latest` is needed )

Reply:
(ah sorry this code is just moved around, but maybe this is something that can be fixed as well ;) )

Reply:
Fixed, thank yo for brining it up.